spec/unit/knife/core/bootstrap_context_spec.rb

```diff
@@ -97,10 +97,10 @@
     end
   end
 
-  describe "when JSON attributes are given" do
-    let(:config) { {:first_boot_attributes => {:baz => :quux}} }
+  describe "when tags are not given" do
```

Comment:
This should be a new test, not a replacement (unless there is a pressing reason to not test the old behavior).